### PR TITLE
Add capacity scaling instructions for UPF to exercises.md

### PR DESCRIPTION
**Pull request by MA3CIN**
I'm not sure why this section was omitted from the exercises.md - I'm willing to iterate on this PR to improve it.

Might need further testing, i encountered an "UPF Cli Run Error - cannot assign requested address" error.

### CODE REVIEW

File: user-guide/exercises.md [hunk 1]
```diff
@@ -897,5 +897,84 @@ In this step, you will change the capacity requirements for the UPF and SMF, and
...
 
-The capacity requirements are captured in a custom resource within the deployed
+The capacity requirements are captured in a custom resource (capacity.yaml) within the deployed
 package.
+
+Start by saving the package Variant downstream target name to a variable.
+
+```
+kubectl config use kind-kind
+upf_packagevariant=$(kubectl get packagevariant edge-free5gc-upf-edge01-free5gc-upf -o jsonpath='{.status.downstreamTargets[0].name}')
```
Comment (johnbelamaric) [inline user-guide/exercises.md:907]: thanks for this, it was an oversight that this was omitted from the docs.

I would suggest that for the user flow, we don't bother with the variables, etc. Those are just for the scripts. So, no need to pull it into /tmp, etc. The flow would be similar to what we did for setting up the regional cluster earlier in the exercises. For example, here I would just have the user copy and paste that from the output of a `kubectl get` or `kubectl describe` of the package variant.

We also can simply manually update the capacity.yaml - we don't need to use a kpt function to do it (though if you want to show that, it's OK too).

If we show the SMF capacity change too, then we could even show a UI-based change to those values.

**Comment by MA3CIN** [comment]
Removed due to wrong CLA - as soon as i get this issue resolved, i will reopen this PR with instructions for SMF included

**Comment by johnbelamaric** [comment]
Awesome, thanks, I was wondering if I scared you off 😂

**Pull request closed without merging**